src/yeahml/build/layers/config.py

```diff
@@ -1,197 +0,0 @@
-import inspect
-
-import tensorflow as tf
-
-
-class NOTPRESENT:
-    def __init__(self):
-        pass
-
-
-# standard_args = {"opts": None, "name": None, "logger": None, "g_logger": None}
-
-# TOOD: could this be automated?
-
-
-def return_available_layers():
```

Comment:
unfortunately, this function is still used and so the file should be kept [mark for keep]